v22.2/alter-database.md

```diff
@@ -1,34 +1,1129 @@
+
+{% include enterprise-feature.md %}
+
+If the [primary region](#set-primary-region) fails, the secondary region becomes the new primary region. For more information, see [Secondary regions](multiregion-overview.html#secondary-regions).
+
+{{site.data.alerts.callout_danger}}
+In order to add a secondary region with `ALTER DATABASE ... SET SECONDARY REGION`, you must first set a primary database region with [`SET PRIMARY REGION`](#set-primary-region), or when [creating the database](create-database.html). For an example showing how to add a secondary region with `ALTER DATABASE`, see [Set the secondary region](#set-the-secondary-region).
+{{site.data.alerts.end}}
+
+For examples, see [Manage regions](#manage-regions).
+
+#### Required privileges
+
+To add a secondary region to a database, the user must have one of the following:
+
+- Membership to the [`admin`](security-reference/authorization.html#admin-role) role for the cluster.
+- Either [ownership](security-reference/authorization.html#object-ownership) or the [`CREATE` privilege](security-reference/authorization.html#supported-privileges) for the database.
+
+#### Parameters
+
+Parameter | Description |
+----------|-------------|
+`region_name` | Usually, the [region](multiregion-overview.html#database-regions) being set as the secondary region for this database. E.g., `"ap-southeast-2"`. Allowed values include any (non-primary) region present in `SHOW REGIONS`. |
+
+### `SURVIVE {ZONE,REGION} FAILURE`
+
+`ALTER DATABASE ... SURVIVE {ZONE,REGION} FAILURE` [statement](sql-statements.html) sets the [survival goal](multiregion-overview.html#survival-goals) for a [multi-region database](multiregion-overview.html).
+
+For examples, see [Configure survival goals](#configure-survival-goals).
+
+#### Required privileges
+
+The user must be a member of the [`admin`](security-reference/authorization.html#roles) or [owner](security-reference/authorization.html#object-ownership) roles, or have the [`CREATE` privilege](security-reference/authorization.html#supported-privileges) on the database.
+
+#### Parameters
+
+Parameter | Description |
+----------|-------------|
+`ZONE` | Set the [survival goal](multiregion-overview.html#survival-goals) to zone failure.
+`REGION` | Set the [survival goal](multiregion-overview.html#survival-goals) to region failure.
 
 ## Viewing schema changes
 
 {% include {{ page.version.version }}/misc/schema-change-view-job.md %}
+
+## Examples
+
+### Manage regions
+
+{% include {{page.version.version}}/sql/multiregion-example-setup.md %}
+
+#### Set the primary region
+
+Suppose you want to make the database `movr` a multi-region database.
+
+To add the first region to the database, or to set an already-added region as the primary region, use a [`SET PRIMARY REGION`](#set-primary-region) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr SET PRIMARY REGION "us-east1";
+~~~
+
+~~~
```

Comment:
I'll file a Style issue if we don't already have one. I'd like to keep consistent for now, since the snippet shows what the successful output looks like. Arguably it's the output that should be changed.